### PR TITLE
feat:video-library authentication done

### PR DESCRIPTION
**Pull request by tushargoyal812**
https://user-images.githubusercontent.com/69681896/160855375-ec1374a7-c7ae-402e-8f68-976f4111458c.mp4

### CODE REVIEW

File: src/components/categories/categories.js [hunk 1]
```diff
@@ -10,14 +10,12 @@ export const Categories=()=>{
     return(
         <main className="quizzes-main flex centered mt-3">
             {category.map(item=>(
-                <div className="card quiz-card">
+                <div key={item.id} className="card quiz-card">
                 <div className="card-main-section quiz-card-main">
                     <img src={item.image} className="product-img quiz-img" alt="category-image" />
                     <main className="middle p-1">
```
Comment (vishalg8454) [inline src/components/categories/categories.js:16]: destructuring can be used.

File: src/pages/login/login.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.login-page-btn{
+    padding: 0.5rem 7rem;
+    flex-shrink: 0;
+}
+
+a{
+    text-decoration: none;
+}
```
Comment (sharmayatendra) [inline src/pages/login/login.css]: remove a tag for SPA

File: src/pages/signup/signup.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.signup-page-btn{
+    padding: 0.5rem 7rem;
+    flex-shrink: 0;
+}
+
+a{
+    text-decoration: none;
+}
```
Comment (sharmayatendra) [inline src/pages/signup/signup.css]: same as above

**Review by sharmayatendra — APPROVED** [review]
overall its good
clean code.
file structure is well maintained.

**Review by vishalg8454 — APPROVED** [review]
Looks good, can merge

File: src/pages/login/login.css (hunk 2 above)
Comment (tushargoyal812) [reply src/pages/login/login.css]: ok will remove it

File: src/pages/signup/signup.css (hunk 3 above)
Comment (tushargoyal812) [reply src/pages/signup/signup.css]: ok

File: src/components/categories/categories.js (hunk 1 above)
Comment (tushargoyal812) [reply src/components/categories/categories.js:16]: ok will do it

**Pull request merged**